src/test/java/step3_racing_car/RacingGameTest.java

```diff
@@ -0,0 +1,68 @@
+package step3_racing_car;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RacingGameTest {
+
+    @DisplayName("운영자가 입력한 경주 대 수 만금 레이싱카가 생성된다.")
+    @Test
+    void countRacingCar(){
+        RacingCars cars = new RacingCars(3);
+        assertThat(cars.racingCars.length).isEqualTo(3);
+    }
+    @DisplayName("초기 생성된 레이싱카의 위치는 0이다")
+    @Test
+    void racingCarDefaultPosition(){
+        RacingCars cars = new RacingCars(3);
+        for (int i = 0; i < cars.racingCars.length; i++) {
+            assertThat(cars.racingCars[i].getPosition()).isEqualTo(0);
+        }
+    }
+
+    @DisplayName("한 레이싱카는 라운드마다 이동 거리만큼 이동한다.")
+    @Test
+    void movingRacingCar(){
+        RacingCar car = new RacingCar();
+        car.movePositions(1);
+        assertThat(car.getPosition()).isEqualTo(1);
+    }
+
+    @DisplayName("모든 레이싱카는 라운드마다 이동 거리만큼 이동한다.")
+    @Test
+    void movingRacingCars(){
+        RacingCars cars = new RacingCars(3);
+        cars.movePosition(1);
+        for (int i = 0; i < cars.racingCars.length; i++) {
+            assertThat(cars.racingCars[i].getPosition()).isEqualTo(1);
+        }
```

Comment:
이렇게 배열 안의 모든 요소를 검증하고 싶을땐 어떤식으로 표현하는게 좋을지 궁금합니다.
이런식으로 반복문을 사용하는 것도 괜찮을까요?

Reply:
```suggestion
        assertThat(cars.racingCars).allSatisfy(car -> assertThat(car.getPosition()).isEqualTo(1));
        assertThat(cars.racingCars).extracting("position").contains(1);
```

반복문을 사용해도 괜찮고 제안드린 2가지 중 1개를 쓰셔도 괜찮습니다.
방법은 많으니 junit 사용법을 한 번 찾아보세요 😎

"테스트 코드가 이해하기 쉬우며 검증해야 할 내용을 검증했는가"가 가장 중요하니 편하신 방법을 사용하시면 돼요.